pkg/converter/sloppy_file_test.go

```diff
@@ -89,3 +93,13 @@ func TestNewSloppyFileInvalidPorts(t *testing.T) {
 		}
 	}
 }
+
+func TestNewComposeFileBuildProperty(t *testing.T) {
+	helper := test.NewHelper(t)
+	cf, err := loadComposeFile("testdata/docker-compose-v2-reject.yml", "no-build-supported")
+	helper.Must(err)
+	_, err = converter.NewSloppyFile(cf)
+	if err == nil {
```

Comment:
This looks like it should also test that the error, when present, has the right message.

Reply:
Could :), I will add the err string comparison